webTestApp/load.js

```diff
@@ -0,0 +1,82 @@
+import http from 'k6/http';
+import {check, sleep} from 'k6';
+import {getRandomPathsQuery} from "./getRandomPathsQuery.js";
+
+/**
+ * 부하 테스트(loadTest)
+ * 평균, 초대 트래픽 같은 적절한 부하를 발생시켜 정상적으로 서버가 동작하는지 확인하여 신뢰성을 얻는다.
+ *
+ *
+ * 테스트 실행법
+ * - 0부터 시작해서 천천히 최대 부하까지 올리기
+ * - 최대에서 다시 0까지 천천히 부하를 내린다.
+ */
+
+/**
+ * @type {Options}
+ */
+export let options = {
+  stages: [
+    { duration: '10m', target: 5 },
+    { duration: '20m', target: 30 },
+    { duration: '10m', target: 0 },
```

Comment:
```suggestion
    { duration: '10s', target: 5 },
    { duration: '10m', target: 5 },
    { duration: '10s', target: 30 },    
    { duration: '20m', target: 30 },
    { duration: '10m', target: 0 },
```

duration 기간만큼 VUser를 늘려가므로, 평소 부하 혹은 피크타임의 부하 값에 대해 테스트하고 싶다면 증가구간를 따로 추가하셔야 합니다.